### PR TITLE
adds center alignment and img responsive class

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

It only adds classes which can be used in projects. There is no css implemented in this plugin.
If someone wants to use center position he has to add styles to his project.

- adds center alignment
- Add img-responsive class if automatic scaling

screen shot 2016-04-27 at 11 22 56

screen shot 2016-04-27 at 14 26 31

### CODE REVIEW

**Comment by czpython** [comment]
BE LGTM

File: README.rst [hunk 1]
```diff
@@ -146,3 +146,9 @@ be used in the template as ``{{ instance.style }}``.
 
 For backwards compatibility the plugin will always use ``cmsplugin_filer_image/image.html`` if it exists. Remove that
 template after migrating to the new structure.
+
+
+Classes
+-------
+
+Classes like left, center, right and img-responsive are given by the plugin to use in your own projects.
```
Comment (vxsx) [inline README.rst]: `left`, `center`, `right` and `img-responsive`

**Comment by vxsx** [comment]
FE LGTM

**Comment by Eraldo** [comment]
Hey nice.. I was just looking for this.
+1 to get this released. :)

**Pull request merged**